indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java

```diff
@@ -1635,6 +1635,7 @@ public Map<String, Object> doGetLiveReports()
     payload.put("ingestionState", ingestionState);
     payload.put("unparseableEvents", events);
     payload.put("rowStats", doGetRowStats());
+    payload.put("recordsProcessed", partitionsThroughput);
```

Comment:
Why not use the `getPartitionStats()` method, same as in `getTaskCompletionReports()`?